docs/spec_draft.md

```diff
@@ -348,6 +349,83 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.batch_norm_inference
+
+### Semantics
+
+Normalizes the `operand` tensor across batch and spatial dimensions, for each
+feature in the `feature_index` dimension and produces a `result` tensor. More
+formally, the semantics can be expressed using Python-like syntax as follows:
```

Comment:
Proposed rewording to: 1) emphasize the relationship with existing ops, 2) to talk about numerical accuracy in the same way that we're doing this with other ops:

```
More formally, this operation can be expressed via a decomposition to existing
StableHLO operations as defined below. Numeric precision is 
implementation-defined.
```